diff --git a/__test__/users.test.tsx b/__test__/users.test.tsx
index daddda9..e87f499 100644
--- a/__test__/users.test.tsx
+++ b/__test__/users.test.tsx
@@ -37,7 +37,7 @@ describe("[POST] /api/users", () => {
       address: "123 Main Street",
       phone_number: "1234567890",
     };
-    await makeHTTPRequest(userHandler, "/api/users/", undefined, "POST", body, StatusCodes.CREATED , body);
+    await makeHTTPRequest(userHandler, "/api/users/", undefined, "POST", body, StatusCodes.CREATED, body);
   });
 
   it("doesn't create a duplicate user", async () => {
@@ -190,8 +190,16 @@ describe("[GET] /api/staff", () => {
     await makeHTTPRequest(staffHandler, "/api/staff", undefined, "GET", undefined, StatusCodes.ACCEPTED, expected);
   });
 
-  it("look for all staff when there are none", async () => {
-    client.query("DELETE from users");
-    await makeHTTPRequest(staffHandler, "/api/users/1", undefined, "GET", undefined, StatusCodes.ACCEPTED, []);
-  });
+
 });
+
+describe("[GET] /api/staff with no staff", () => {
+  beforeAll(async () => {
+    await client.query("DELETE from users WHERE role = 'Teacher' OR role = 'Admin'")
+  })
+
+  it("Works correctly with 0 users", async () => {
+    await makeHTTPRequest(staffHandler, "/api/users/", undefined, "GET", undefined, StatusCodes.ACCEPTED, []);
+  });
+
+})
